app/experimenter/templates/experiments/section_base.html

```diff
@@ -22,12 +22,15 @@ <h4 id="{{ section_name }}" class="col section-title">
             {% block section_content %}
               {{ section_content|urlize|linebreaks }}
             {% endblock %}
+          {% elif section_name == "results" and not section_complete %}
+            <p>Please <a href="{% url edit_url_name slug=experiment.slug %}">add results </a> to this experiment.</p>
+
           {% else %}
             <p>You must <a href="{% url edit_url_name slug=experiment.slug %}">complete the {{ section_name }} form</a> before you can launch your experiment.</p>
           {% endif %}
         </div>
 
-        {% if experiment.is_editable  and edit_url_name %}
+        {% if experiment.is_editable  and edit_url_name or experiment.is_begun and section_name == "results" %}
```

Comment:
Sigh, these django template includes aren't great for extensibility.  There probably isn't a better way to do this right now, but let's keep this special case in mind when we start refactoring the front end.